testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/SseBroadcastTest.java

```diff
@@ -175,6 +172,8 @@ public void testBroadcasterOnCloseCallbackCloseSinkOnServer() throws Exception {
                 logger.error(ex.getMessage(), ex);
             });
             eventSource.open();
+            //Let's wait for SseEventSource to be connected
+            TimeUnit.SECONDS.sleep(3);
```

Comment:
Without this sleep time, invocations of  `client.target(generateURL("/broadcast/startAndClose")).request()
.post(Entity.entity(textMessage, MediaType.SERVER_SENT_EVENTS))` will throw a NPE on server side and will not try to broadcast message since `this.eventSink` will be null because of no SseEventSINK connected yet